examples/webassembly/wasm_simple_add.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
```

Comment:
Move `examples/webassembly/wasm_simple_add.py` to `examples/python/wasm_simple_add.py`.

The `python` means using the python client sdk.

Reply:
Also add this to the `run_examples` in `cmake/scripts/test.sh`.